commec/__init__.py

```diff
@@ -0,0 +1,5 @@
+from importlib.metadata import version
+try:
+    __version__ = version(__name__)
+except:
+    __version__ = "X.X.X"
```

Comment:
I think directly referencing the package name is preferable (just in case this is imported in a weird way_

```suggestion
from importlib.metadata import version, PackageNotFoundError
try:
    __version__ = version("commec")
except ImportError, PackageNotFoundError:
    __version__ = "X.X.X"

```